usr/lib/systemd/system/proxysql.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=High Performance Advanced Proxy for MySQL
+
+[Service]
+Type=forking
+PIDFile=/var/lib/proxysql/proxysql.pid
+ExecStart=/usr/local/bin/proxysql
```

Comment:
I am not very familiar with systemd (yet).
Would it be possible to set options and arguments like in initd scripts?
I would prefer to explicitly pass config file and datadir